gas/testsuite/gas/riscv/zc-norm-cm-relax-immu.d

```diff
@@ -1,4 +1,4 @@
-#as: -march=rv32g_zcmb
+#as: -march=rv32g_zcb
 #source: zc-norm-cm-relax-immu.s
 #objdump: -d -Mno-aliases
 
```

Comment:
zc-norm-cm-relax-immu.[s|d] test now looks to be duplicated with zc-zcb* tests. Suggest removing them.

Reply:
@linsinan1995 They are not quite are the same.  'zc-norm-cm-relax-immu.s' uses the canonical names of the instructions (i.e. `lhu` rather than `c.lhu`, and tests that he assembler relaxes down to the Zc* versions where possible (not possible for unaligned half words).  The 'zc-c-relax-fail.s' and 'zc-c-relax-pass.s' tests explicitly specify the Zc* versions of instructions, and verify that unaligned half word access fails.